### PR TITLE
[MOSIP-28755] added clearcache and clear unused images yaml to env-ch…

### PR DESCRIPTION
**Pull request by Rakshitha650**
…eck.yaml

### CODE REVIEW

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: mosip/on-prem/env-check.yaml [hunk 1]
```diff
@@ -17,3 +17,17 @@
...
+        name: "check dirs"
+        minute: "0"
+        hour: "0"
+        job: "sync; echo 1 > /proc/sys/vm/drop_caches"
+
+    - name: clear unused docker images
+      ansible.builtin.cron:
+        name: "check dirs"
+        minute: "0"
+        hour: "0"
```
[the remark this answers is not in the document]
Comment (Rakshitha650) [reply mosip/on-prem/env-check.yaml:32]: done the same

File: mosip/on-prem/env-check.yaml [hunk 2]
```diff
@@ -17,3 +17,17 @@
       fail:
        msg: Localhost mapping does not exist.
       when: my_var == ""
+
+    - name: clear cache
+      ansible.builtin.cron:
+        name: "check dirs"
```
Comment (syedsalman3753) [inline mosip/on-prem/env-check.yaml:23]: please change name from "check dirs" to "remove memory cache"

File: mosip/on-prem/env-check.yaml [hunk 3]
```diff
@@ -17,3 +17,17 @@
...
+    - name: clear cache
+      ansible.builtin.cron:
+        name: "check dirs"
+        minute: "0"
+        hour: "0"
+        job: "sync; echo 1 > /proc/sys/vm/drop_caches"
+
+    - name: clear unused docker images
+      ansible.builtin.cron:
+        name: "check dirs"
```
Comment (syedsalman3753) [inline mosip/on-prem/env-check.yaml:30]: please change name from "check dirs" to "remove unused docker images"

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

**Review by syedsalman3753 — APPROVED** [review]

File: mosip/on-prem/env-check.yaml
Comment (ckm007) [inline mosip/on-prem/env-check.yaml:1]: rename to env-check-setup.yaml and add in the readme there the new usage added.

File: mosip/on-prem/env-check.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ckm007) [reply mosip/on-prem/env-check.yaml:32]: will this run every 5 mins?

File: mosip/on-prem/README.md [hunk 4]
```diff
@@ -44,9 +44,9 @@ ansible-playbook -i hosts.ini ports.yaml
 ansible-playbook -i hosts.ini swap.yaml
 ```
 ## Environment Check
```
Comment (ckm007) [inline mosip/on-prem/README.md:46]: Environment Check and setup
1. Performs check to make sure localhost mapping is present in all nodes.
2. Adding these crons:....

**Review by ckm007 — APPROVED** [review]

**Pull request merged**